common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -403,6 +428,14 @@ open class MusicService : MediaBrowserServiceCompat() {
         previousPlayer?.stop(/* reset= */true)
     }
 
+    private fun saveRecentMediaItemId(mediaId: String?) {
```

Comment:
I wonder if you would mind moving this into a class and have these methods be suspend functions? (How much would that break things?)

I ask because it would be nice to be able to change the implementation away from Shared Preferences by just replacing the internal implementation of that one class, and given we are doing I/O, it feels like we shouldn't do it on the main thread.